ci-operator/templates/cluster-launch-installer-e2e.yaml

```diff
@@ -57,6 +59,45 @@ objects:
       secret:
         secretName: ${JOB_NAME_SAFE}-cluster-profile
 
+    initContainers:
+    - name: share-installer-src
+      image: registry.svc.ci.openshift.org/${NAMESPACE}/pipeline:src
+      volumeMounts:
+      - name: shared-tmp
+        mountPath: /tmp
+      command:
+      - cp
+      - -R
+      - .
+      - /tmp/installer-src
+
+    - name: bazel-build-smoke
+      image: quay.io/coreos/tectonic-builder:bazel-v0.3
+      volumeMounts:
+      - name: shared-tmp
+        mountPath: /tmp
+      - name: artifacts
+        # bazel needs to write to .cache
+        mountPath: /.cache
+      command:
+      - /bin/bash
+      - -c
+      - |
+        #!/bin/bash
+        set -euo pipefail
+        echo "ls -al /tmp/src-installer"
+        # wait for source code to show up
+        if [[ ! -d /tmp/installer-src ]]; then
+          echo "Waiting for installer source code to show up ..."
+          sleep 10 & wait
+        fi
```

Comment:
@stevekuznetsov I'm following suit here with `sleep 10 & wait` but not sure what's the difference than `sleep 10`  and also, probably don't need that at all, since I believe init containers run 1 at a time, sequentially? If you confirm that, I'll remove

Reply:
Correct, `initcontainers` run sequentially

Reply:
ok, I'll remove that